### ISSUE TITLE
Update Buff to BFA's ones ( Report )

### ISSUE THREAD

**Issue by CXWRTH** [Lumiahna/RaidBuffStatus_Continued#6]
I haven't tested all, but it appears the spell IDs might have changed since RBS was popular?

http://www.wowhead.com/news=281692/battle-for-azeroth-class-raid-buff-abilities-and-dispel-utility

I tested with a warrior that had Battle Shout and RBS was not reporting buff status on group for it.

**Comment by Lumiahna** [comment]
Yes, the IDs changes, even the Buff on some class, It'll be updated when I'll be able to fix the error you reported me earlier #5

**Comment by CXWRTH** [comment]
<3 Thank you for your hard work.
